example.env

```diff
@@ -4,7 +4,7 @@ MODULE_NAME=i2c-input
 MODULE_TYPE=Input
 I2C_INTERFACE_NUMBER=1
 DATA_TYPE=byte
-SLAVE_ADDR=10
+SLAVE_ADDR=104
 PERIOD=1
```

Comment:
this needs to be changed to milliseconds I assume, also in other places where you define default values

Reply:
the period could be arbitrarily 1000ms (that is why no matter to use that value as default value)

Reply:
but rn your default value in 1ms. do you want to wait for 1ms?